src/WorkBC.Data/Migrations/20240413000723_InsertNocCodes2021.cs

```diff
@@ -0,0 +1,136 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System.Text.Json;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System;
+using WorkBC.Data.Model.JobBoard;
+using Microsoft.EntityFrameworkCore.Metadata.Internal;
+using WorkBC.Data.Model.Enterprise;
+using System.IO;
+using Microsoft.VisualBasic;
+using System.Text;
+using System.Linq;
+
+#nullable disable
+
+namespace WorkBC.Data.Migrations
+{
+    public partial class InsertNocCodes2021 : Migration
+    {
+        private readonly HttpClient _httpClient;
+        private readonly string _ssotUrl;
+
+        public InsertNocCodes2021() : base()
+        {
+            _httpClient = new HttpClient();
+            _ssotUrl = Environment.GetEnvironmentVariable("SSOT_URL");
+            if (_ssotUrl == null)
+            {
+                throw new Exception("SSOT_URL not set");
+            }
+        }
+        protected async override void Up(MigrationBuilder migrationBuilder)
+        {
+            Console.WriteLine("Hello");
```

Comment:
Please remove this debug line.

Reply:
Removed all debug console commands in the latest commit.